data/spells/scripts/support/find_person.lua

```diff
@@ -46,12 +46,7 @@ function onCastSpell(creature, variant)
 	end
 
 	local playerPosition = creature:getPosition()
-	local targetPosition = targetPlayer:getPosition()
-	local offset = {
-		x = playerPosition.x - targetPosition.x,
-		y = playerPosition.y - targetPosition.y,
-		z = playerPosition.z - targetPosition.z
-	}
+	local offset = playerPosition - targetPlayer:getPosition()
```

Comment:
That won't work;
21:19 Druid is to the west.
21:19 Druid is very far to the west.

First one im standing to the east, second one im standing to the west